tests/test_micropipenv.py

```diff
@@ -253,6 +253,25 @@ def test_install_pip_tools_editable(venv):
             shutil.rmtree("micropipenv_editable_test.egg-info", ignore_errors=True)
 
 
+@pytest.mark.skipif(PIP_VERSION.release >= (19, 3, 0), reason="Direct reference installation is supported in pip starting 19.3")
```

Comment:
```suggestion
@pytest.mark.skipif(PIP_VERSION.release < (19, 3, 0), reason="Direct reference installation is supported in pip starting 19.3")
```

Reply:
The test is skipped if the condition is True.

Reply:
Thanks :smiley: